test/browser/performance.js

```diff
@@ -21,6 +21,7 @@ function loop(iter, time) {
 
 
 function benchmark(iter, callback) {
+	/* eslint-disable no-unused-vars */
```

Comment:
`eslint-disable` is not block-scoped, so this applies to the entire file from this line on rather than just this function. You can scope the lint ignore to just the `a` variable using:

```
let a = 0; // eslint-disable-line no-unused-vars
```